test/e2e/parallel/space_test.go

```diff
@@ -506,19 +506,28 @@ func TestSubSpaceInheritance(t *testing.T) {
 		// when
 		// we have a parentSpace
 		t.Logf("Create parent space")
-		parentSpace, _, parentSpaceBindings := CreateSpace(t, awaitilities, testspace.WithSpecTargetCluster(memberAwait.ClusterName), testspace.WithTierName("appstudio"))
-		// then
-		// wait until MUR and Space have been provisioned
-		t.Logf("Wait for space to be provisioned")
-		VerifyResourcesProvisionedForSpace(t, awaitilities, parentSpace.Name, UntilSpaceHasStatusTargetCluster(memberAwait.ClusterName))
-		_, err = hostAwait.WaitForMasterUserRecord(t, parentSpaceBindings.Spec.MasterUserRecord)
-		require.NoError(t, err)
+		parentUser := NewSignupRequest(awaitilities).
+			ManuallyApprove().
+			TargetCluster(memberAwait).
+			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+			SpaceTier("appstudio").
+			EnsureMUR().
+			Execute(t)
+
+		user := NewSignupRequest(awaitilities).
+			ManuallyApprove().
+			TargetCluster(memberAwait).
+			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
+			SpaceTier("appstudio").
+			EnsureMUR().
```

Comment:
to speed up the logic - we don't need to verify all the resources provisioned for MUR and Space
```suggestion
			WaitForMUR().
```